src/main/java/com/targomo/client/api/json/AbstractLocationMapDeserializer.java

```diff
@@ -0,0 +1,35 @@
+package com.targomo.client.api.json;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.targomo.client.api.geo.Location;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Gideon Cohen
+ */
+
+public abstract class AbstractLocationMapDeserializer<K extends Location> extends JsonDeserializer<Map<String, K>> {
+
+    private ObjectMapper mapper = new ObjectMapper();
```

Comment:
this should be a `static final` constant

Reply:
Updated